packages/client/src/__tests__/dmmfTypes.test.ts

```diff
@@ -23,7 +23,7 @@ model User {
 }
 `
 
-test('dmmf types', async () => {
+test.skip('dmmf types', async () => {
```

Comment:
```suggestion
// Skip because it cannot work anymore via @swc/jest
// TODO: Port this test to the new functional test setup
test.skip('dmmf types', async () => {
```